common/rfb/EncodeManager.cxx

```diff
@@ -550,7 +550,7 @@ Encoder *EncodeManager::startRect(const Rect& rect, int type)
   encoder = encoders[klass];
   conn->writer()->startRect(rect, encoder->encoding);
 
-  if (encoder->flags & EncoderLossy)
+  if (encoder->flags & EncoderLossy && !encoder->treatLossless())
```

Comment:
This is an entirely different feature, so let's keep that separate. This PR will get complex enough as it is. :)